tests/test_snql_api.py

```diff
@@ -549,3 +550,121 @@ def test_nullable_query(self):
             ),
         )
         assert response.status_code == 200
+
+    def test_invalid_column(self) -> None:
+        response = self.post(
+            "/discover/snql",
```

Comment:
Outcomes queries should go to `/outcomes/snql` not `/discover/sqnl` -> it works because datasets are never checked today, we just honour whatever entity is asked for in the `MATCH` clause, but shouldn't really. Same for the other tests in this file.
```suggestion
            "/outcomes/snql",
```